.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+quote_type = double
+
+[*.kt]
+indent_size = unset
+quote_type = unset
```

Comment:
What was the reason for unsetting those values? :)

Reply:
I was not sure anymore if the quotes are different due to ktlint. It's not necessary to define it here, as ktlint is going to complain about them anyway, if they are wrong.